go/oasis-test-runner/oasis/oasis.go

```diff
@@ -55,6 +57,8 @@ type Node struct {
 
 	restartable bool
 	doStartNode func() error
+
+	LogWatcherHandlerFactories []log.WatcherHandlerFactory
```

Comment:
Group public fields together? Also do these need to be publicly mutable (probably not?) or can they be private and just have public accessor methods?

Reply:
Yes, they don't need to be publicly mutable. I've renamed them to `logWatcherHandlerFactories`.